extensions/wikia/Privacy/RemoveUserDataController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+use Wikia\Logger\Loggable;
+
+class RemoveUserDataController extends WikiaController {
+	use Loggable;
+
+	const ACCEPTED = 202;
+	const METHOD_NOT_ALLOWED = 405;
+
+	public function allowsExternalRequests() {
+		return false;
+	}
+
+	public function removeUserData() {
```

Comment:
Should we guard access to this method via internal request check or a token?

Reply:
`allowsExternalRequests` guard should prevent it